waspc/data/Generator/templates/server/package.json

```diff
@@ -16,7 +16,8 @@
     "delay": "1000"
   },
   "engines": {
-    "node": ">={= nodeVersion =}"
+    "node": "^{= nodeVersion =}",
+    "npm": "^{= npmVersion =}"
```

Comment:
The caret `^` stands for "semantically compatible with". For example, `^16.0.0` allows all version that look like `16.x.x`

Check [NPM's docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies) for details.